static/app/components/organizations/pageFilters/desyncedFiltersAlert.tsx

```diff
@@ -76,14 +76,14 @@ export default function DesyncedFilterAlert({
         </Fragment>
       }
     >
-      {tct(
-        message ??
+      {message ??
```

Comment:
I checked that message wasn't inserting any of the `filter` or `has` parameters.